src/client/components/app/Navbar/index.tsx

```diff
@@ -74,6 +83,7 @@ const getNetworkIcon = (network: Network) => {
 interface NavbarProps {
   className?: string;
   title?: string;
+  vaultName?: string;
```

Comment:
Maybe use a more generic prop name for this one, so it can be reused for non-vault related page in the future.

Reply:
changed it to subtitle